[DROOLS-6736] Add kie-server docs for container alias (#4114)
kiegroup/kie-docs: doc-content/shared-kie-docs/src/main/asciidoc/KieServer/Creation-section.adoc

The figure above shows that there are three deployable packages available to be used as containers on the Execution Server.
Select the one that you want by clicking the Select button.
This will auto-populate the *GAV* and you can then click the *Ok* button to use this deployable as the new Container.

> . Optionally, enter an alias for this Container, which you can use instead of its container name when you access the Execution Server. By default, the alias is its *Artifact Id*.

Reviewer: ```suggestion
. Optionally, enter an alias for this Container, which you can use instead of the container name while accessing the execution server. By default, the alias is the related *Artifact Id*.
```

+

[NOTE]
====

> You can configure the same alias for different containers. When you use the alias, the latest version of associated containers will be executed. It would be convenient when you deploy a new container version but you don't want to change client codes.

Reviewer: ```suggestion
You can configure the same alias for different containers. When you use the alias, the latest version of associated containers is executed. This can be helpful when you deploy a new container version but do not want to change the client codes.
```
